build/yamls/base/crds.yml

```diff
@@ -34,3 +34,21 @@ spec:
     kind: AntreaAgentInfo
     shortNames:
       - aai
+---
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: traceflows.antrea.tanzu.vmware.com
+spec:
+  group: antrea.tanzu.vmware.com
```

Comment:
I think Traceflow is ok to be under antrea since it is not like agentinfo and controllerinfo. If we have other crds similar to Traceflow and then we can add later?

Reply:
I don't have strong opinion here, but I looked at K8s apis, all resources except those under core group have a separate group name, so I would suggest to follow it in case of any unknown consequence that may affect API evolution.
I don't know a proper group name, but we may use "traceflow.antrea.tanzu.vmware.com" as the group name until anyone think of a better one, given "Event" resource is also in "events.k8s.io" group: https://github.com/kubernetes/kubernetes/blob/15e95e48964d4e800f7210a249f9122ee7d79fcc/pkg/apis/events/register.go

Reply:
Agreed. Use "traceflow.antrea.tanzu.vmware.com".

Reply:
So we don't believe that we will have other CRDs that should be logically grouped with the traceflow one?

Reply:
How about ops.antrea.tanzu.vmware.com, or diagnostics.antrea.tanzu.vmware.com?

Reply:
I prefer "ops" one. How do you like it? @tnqn

Reply:
sounds good to me

Reply:
Updated.